collection-scripts/gather_network_observability

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NO_NS="$(oc get subs -A -o template --template '{{range .items}}{{if eq .spec.name "netobserv-operator"}}{{.metadata.namespace}}{{end}}{{end}}')"
+
+if [ -z "${NO_NS}" ]; then
+    echo "INFO: Network Observability namespace is not detected. Skipping."
+    exit 0
+fi
+
+function get_netobserv_cluster_crs() {
+    CRD="flowcollectors.flows.netobserv.io"
+    oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "${CRD}"
+    obj=$(oc get ${CRD} -o custom-columns=:metadata.name --no-headers)
+    FC_NS=$(oc get ${CRD} ${obj} --output=jsonpath='{.spec.namespace}')
+}
+
+oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "ns/${NO_NS}"
+
+get_netobserv_cluster_crs
```

Comment:
Why are we doing this vs just calling `oc admin inspect` on the cluster resource and letting [inspect](https://github.com/openshift/oc/blob/master/pkg/cli/admin/inspect/resource.go#L51) resolve the resources for the operator (including any CRD's that it defines)? 

@deads2k we may not be able to do this for OLM operators; but if that's true what can we do to fix that (we may have had this conversation before).